query-connector/src/app/(pages)/userManagement/userGroups/page.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { useContext, useEffect, useState } from "react";
+import classNames from "classnames";
+import { Button } from "@trussworks/react-uswds";
+import Table from "../../../ui/designSystem/table/Table";
+import { UserGroup, userGroupsMock } from "../types";
+import { DataContext } from "../components/DataProvider";
+
+/**
+ * User groups section in the user management page
+ * @returns The user groups table
+ */
+const UserGroups: React.FC = () => {
+  const [userGroups, setUserGroups] = useState<UserGroup[]>([]);
+
+  const { OpenEditSection } = useContext(DataContext);
```

Comment:
super annoying nit: is there any rule for us using CapitalizedCamelCase like we have here rather than lowerCaseCamelCase? This just jumped out at me readability wise. 

Obviously don't want to spend too much time here, but maybe worth setting up some linting to keep the function names consistently styled